src/configs/cmd_duration.rs

```diff
@@ -24,6 +25,7 @@ impl<'a> Default for CmdDurationConfig<'a> {
             disabled: false,
             show_notifications: false,
             min_time_to_notify: 45_000,
+            notification_timeout: 750,
```

Comment:
As long as we're here, we don't have to keep the default notification length.

You noted in the original PR that 750ms is too short--what's the minimum value that you think is long enough to be usable?

Reply:
And if you're happy with this or don't have a suggestion, let me know and I'll approve.

Reply:
Another option might be to change this to `Option<u32>` (if so, don't forget adding ` #[serde(skip_serializing_if = "Option::is_none")]` to the struct item) and map `None` to `Timeout::Default`.

Reply:
A default of 2 seconds seems reasonable.

Using an Option to make the default Timeout:: Default is something I haven't looked at. I like the idea and it is worth considering. I'll see what I can implement.